task2.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+    "fmt"
+    "sort"
+)
+
+type Point struct {
+  x, y int
+}
+
+type Square struct {
+  start Point
+  a uint
+}
+
+func (s Square) End() Point {
+  p := Point{x: s.start.x + int(s.a), y: s.start.y - int(s.a)}
```

Comment:
Variable 'p' is redundant, just write `return Point{x: s.start.x + int(s.a), y: s.start.y - int(s.a)}`

Reply:
Thanks, i had some wierd issues about that method. Just fixed it and pushed here.